did-doc-diff/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "@adxp/did-doc-diff",
```

Comment:
this is fine for now 👍 

im guessing we'll want to add more to this pacakge than just diffs. so something like: `@adxp/did-aic`